server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -238,6 +261,19 @@ async function verifyToken(
 		}
 	}
 
+	if (tokenCacheEnabled && tokenCache) {
```

Comment:
lines 264, 277, 252-255 can be removed. verifyToken already has the logic corresponding to these lines.